upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -18,88 +18,86 @@ package awstasks
 
 import (
 	"fmt"
-
 	"reflect"
 	"sort"
 	"strings"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/autoscaling"
-	"github.com/golang/glog"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/cloudup/awsup"
 	"k8s.io/kops/upup/pkg/fi/cloudup/cloudformation"
 	"k8s.io/kops/upup/pkg/fi/cloudup/terraform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/golang/glog"
 )
 
 const CloudTagInstanceGroupRolePrefix = "k8s.io/role/"
 
+var (
+// terminationPolicies is the termination policy of a ASG
+//terminationPolicies = []string{"OldestInstance", "OldestLaunchConfiguration"}
+)
+
 //go:generate fitask -type=AutoscalingGroup
 type AutoscalingGroup struct {
 	Name      *string
 	Lifecycle *fi.Lifecycle
 
-	MinSize *int64
-	MaxSize *int64
-	Subnets []*Subnet
-	Tags    map[string]string
-
-	Granularity *string
-	Metrics     []string
-
+	Granularity         *string
```

Comment:
I appreciate you need for order :) But if I am reading this diff correctly the actual code changes are really small.  If you can just comment on the code change, it will make the life of a reviewer much simple.